src/index.ts

```diff
@@ -1,3 +1,5 @@
 export { getExtensions, watchExtensions, initializeKiltExtensionAPI } from './getExtension/getExtension'
-import * as Types from './types/types'
+export * as WellKnownDidConfiguration from './wellKnownDidConfiguration/wellKnownDidConfiguration'
```

Comment:
I think @weichweich has put in some effort to remove this export from here to have the library usable for the frontend code

Reply:
introduced subpath exports to deal with this, what do you think?

Reply:
All the formatting changes make it difficult to follow.
Why does `getExtension` gets its own sub-path? Why do `types`?  Personally, I think that only `wellKnownDidConfiguration` absolutely must be on a separate path.

Reply:
for consistency. The exports map will prevent supporting implementations from importing anything that is not in the map, so while non-supporting applications could use the types subpath (based simply on the file system path), exports supporting applications couldn't

Reply:
ultimately we'd probably want `frontend`, `backend` and `extension` paths though, I'd assume?

Reply:
the formatting changes are the result of someone adding an .editorconfig, which changes how the code is auto-formatted, sorry for that. Should have been done together with adding the config.

Reply:
Here’s my take on this. The export paths are part of the API of the package. The API should be minimal for multiple obvious reasons. The existence of a technical possibility for someone to import things beyond the official API is not a reason to add more things to the API (an attempts to lock things down never work well).

Reply:
getExtension is definitely debatable, for the types I'd rather remove the export from the main index though, because using a type via 
```ts
import { Types } from 'kilt-extension-api'
const something: Types.Something = { ... }
```
is super ugly.

Reply:
It is super ugly, that’s why people don’t do `import * as Types; export { Types }` but rather `export * from types`.